addons/nametags/stringtable.xml

```diff
@@ -240,18 +240,21 @@
             <Polish>Wymuś pod kursorem</Polish>
             <Spanish>Forzar mostrar solo en el cursor</Spanish>
             <Czech>Vynuceno zobrazit pouze na kurzor</Czech>
+            <German>Erzwinge nur mit Cursor anzuzeigen</German>
```

Comment:
Ich bin mir nicht im klaren wie es in anderen Regionen Deutschlands ist, aber bei uns ist der Begriff Mauszeiger geläufiger.